tests/virt_autotest/libvirt_host_bridge_virtual_network.pm

```diff
@@ -53,7 +53,9 @@ sub run_test {
     my $gi_host_bridge = '';
     foreach my $guest (keys %xen::guests) {
         record_info "$guest", "HOST BRIDGE NETWORK for $guest";
-        assert_script_run("virsh attach-interface $guest network vnet_host_bridge --live");
+        #figure out that used with virtio as the network device model during
+        #attach-interface via virsh worked for all sles guest
+        assert_script_run("virsh attach-interface $guest network vnet_host_bridge --model virtio --live");
```

Comment:
Does virtio model work for all products, especially sle11sp4?

Reply:
My suggestion is that if only 15sp2+ needs to specify this virtio model, then you can add this  special operation for virtio only to 15sp2+ host  by a if condition and keeps old products using default. Then no potential issue for  11sp4.

Reply:
As I understood, virtio model work for all products, especially sle11sp4. Now, I am working on checking with this on sles11sp4 host. however, there was really and really low network..... on my side today.

Reply:
Quick update, I had confirmed that virtio model work very well on sles11sp4 host with sles15sp2 guest.
as below:
# virsh net-list --all
 Name                 State      Autostart     Persistent
----------------------------------------------------------
 vnet_host_bridge     active     no            no
# virsh domiflist sles-15-sp2-64-fv-def-net
Interface  Type       Source     Model       MAC
-------------------------------------------------------
# virsh attach-interface sles-15-sp2-64-fv-def-net network vnet_host_bridge --model virtio --live
Interface attached successfully
# virsh domiflist sles-15-sp2-64-fv-def-net
Interface  Type       Source     Model       MAC
-------------------------------------------------------
vnet0      bridge     br0        virtio      52:54:00:4a:3a:1a
# cat /etc/os-release
NAME="SLES"
VERSION="11.4"
VERSION_ID="11.4"
PRETTY_NAME="SUSE Linux Enterprise Server 11 SP4"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:11:4"

So, make sure that virtio model work for all sles products.
Also, current code changed in this PR work for all sles products.

Reply:
Great! Merge it